tensorflow/compiler/xla/service/gpu/cublas_cudnn.cc

```diff
@@ -107,5 +133,40 @@ std::string CudnnConvKindToString(CudnnConvKind kind) {
   }
 }
 
+StatusOr<CudnnfMHAKind> GetCudnnfMHAKind(
+    const HloCustomCallInstruction* instr) {
+  absl::string_view target = instr->custom_call_target();
+  if (target == kCudnnfMHABmmBmmCallTarget) return CudnnfMHAKind::kBmmBmm;
```

Comment:
Nit: braceless ifs are generally avoided in the codebase, as adding logging statements can lead to goto-fails.

Reply:
Done.